product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx

```diff
@@ -25,24 +25,24 @@ At the end of each quick starts, you'll have a cluster with four nodes and these
 You'll use these hostnames throughout this exercise.
 
 !!! Note A best practice recommendation
-This example is based on the quick started configuration and, for speed
-and simplicity, it uses the barman backup server in place of creating a bastion
-server. It also uses the barman login to the Postgres cluster. 
+This example is based on the quick start configuration. For speed
+and simplicity, it uses the Barman backup server in place of creating a bastion
+server. It also uses the Barman login to the Postgres cluster. 
 
-In a production environment, it is recommended that you create a separate
-bastion server to run the failover experiment from and to create an appropriate
-Postgres user to log into the cluster. 
+In a production environment, we recommend that you create a separate
+bastion server to run the failover experiment from and that you create an appropriate
+Postgres user to log in to the cluster. 
 !!!
 
 ## Installing xpanes
 
 !!! Note Xpanes optional
-We recommend the `xpanes` utility for this exercise. It allows you to easily switch between multiple terminal sessions. If you prefer to use multiple terminals, tmux or another terminal multiplexer, you can do so. Just make sure you can easily switch between multiple terminal sessions.
+We recommend the xpanes utility for this exercise. It allows you to easily switch between multiple terminal sessions. If you prefer to use multiple terminals, tmux, or another terminal multiplexer, you can do so. Just make sure you can easily switch between multiple terminal sessions.
 !!!
 
-You'll be using `xpanes`, a utility that allows you to quickly create multiple terminal sessions that you can easily switch between. It isn't installed by default, so you'll have to install it. For this exercise, you'll be launching xpanes from the system where you ran tpaexec to configure your quick-start cluster.
+You'll use xpanes, a utility that allows you to quickly create multiple terminal sessions that you can easily switch between. It isn't installed by default, so you have to install it. For this exercise, you launch xpanes from the system where you ran tpaexec to configure your quick-start cluster.
```

Comment:
Could we add some guidance (and examples) around this in the Style Guide? https://www.enterprisedb.com/docs/community/contributing/styleguide/#inline-code 

I would have used the in-line code ticks here and for psql earlier on this PR and now I am not sure about the rule.  What about utilities like `initdb`?

Reply:
Utility names don't get the ticks. I think that's clear in the style guide but I'll double check.